docs/GettingStarted/GeneratingData.md

```diff
@@ -96,7 +96,7 @@ The manifest shows which rules are violated in which file.
 ## Hints and Tips
 
 * The generator will output velocity and row data to the console as standard
-(see [options](../Options/GenerateOptions.md) for other monitoring choices).
+(see [options](../Options/GenerateOptions.md) for other monitoring choices)
```

Comment:
I suspect the better option would be to add missing full stops on the other lines, not remove the ones that are already here, given that that's what you have done elsewhere in this PR